gammapy/astro/population/simulate.py

```diff
@@ -126,21 +138,24 @@ def make_catalog_random_positions_sphere(size, center='Earth', distance=Quantity
     return table
 
 
-'''
 def make_cat_gauss_random(n_sources=100, glon_sigma=30, glat_sigma=1,
```

Comment:
This function was placed between 2 sets of triple single quotes. Why? Was it an bug? Was the function commented out for some reason? Or are these quotes this some kind of decorator?